### PR TITLE
feat(npm-lit): add mixin for Lit

### PR DESCRIPTION
**Pull request by kaifaty**
Add mixin for Lit

### CODE REVIEW

File: packages/core/src/atom.ts [hunk 1]
```diff
@@ -200,9 +201,9 @@ export const createCtx = ({
   callLateEffect = callSafely,
   callNearEffect = callSafely,
 }: CtxOptions = {}): Ctx => {
-  let caches = new WeakMap<AtomProto, AtomCache>()
-  let read = (proto: AtomProto): undefined | AtomCache => caches.get(proto)
-  let logsListeners = new Set<Fn<[Logs, Error?]>>()
+  const caches = new WeakMap<AtomProto, AtomCache>()
```
Comment (artalar) [inline packages/core/src/atom.ts]: What does this for? Move it to separate PR pls

File: packages/npm-lit/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# What is Lit?
+
+Lit is a simple library for building fast, lightweight web components.
+
+At Lit's core is a boilerplate-killing component base class that provides reactive state, scoped styles, and a declarative template system that's tiny, fast and expressive
+
+## About
+
+Package provide `reatomLit` static class with 2 methods:
```
Comment (artalar) [inline packages/npm-lit/README.md]: Outdated

File: packages/npm-lit/package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    "test": "tsx src/index.test.ts",
+    "test:watch": "tsx watch src/index.test.ts"
+  },
+  "dependencies": {
+    "@reatom/async": ">=3.1.0",
+    "@reatom/utils": ">=3.1.0",
+    "lit": "^2.6.0",
+    "@reatom/core": ">=3.1.0"
+  },
+  "author": "artalar",
```
Comment (artalar) [inline packages/npm-lit/package.json]: Did you explicitly remove yourself or just forget?

```suggestion
  "author": "kaifaty",
  "maintainers": [
    {
      "name": "artalar",
      "url": "https://github.com/artalar"
    },
    {
      "name": "kaifaty",
      "url": "https://github.com/kaifaty"
    }
  ],
```

File: packages/npm-lit/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,102 @@
+import type {Ctx, CtxSpy, Atom, Unsubscribe} from '@reatom/core'
+import {atom, throwReatomError} from '@reatom/core'
+import {nonNullable} from '@reatom/utils'
+import {LitElement, PropertyValues} from 'lit'
+
+
+type Constructor<T> = new (...args: any[]) => T;
+
+const compare = (a: unknown[], b: unknown[]) => {
```
Comment (artalar) [inline packages/npm-lit/src/index.ts]: use `isShallowEqual` or short this

```suggestion
const isShallowEqual = (a: unknown[], b: unknown[]) => a.length === b.length && a.every((el, i) => el === b[i])
```

**Review by artalar — CHANGES_REQUESTED** [review]

File: packages/core/src/atom.ts (hunk 1 above)
Comment (kaifaty) [reply packages/core/src/atom.ts]: Sorry, don't want to move it to PR. Just find out what variables are mutatable.

File: packages/npm-lit/src/index.ts (hunk 4 above)
Comment (kaifaty) [reply packages/npm-lit/src/index.ts]: ok, but: 
изображение

Comment (artalar) [reply packages/npm-lit/src/index.ts]: Trust me, this is **nothing** :)

File: packages/npm-lit/README.md (hunk 2 above)
Comment (artalar) [reply packages/npm-lit/README.md]: @kaifaty fix docs pls

File: packages/npm-lit/src/index.ts [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+
+/**
+ * Mixin subscribes for atoms used in component render
+ * 
+ * @param superClass - LitElement extended class
+ * 
+ * @returns class with ctx property
+ */
+export const reatomMix = <T extends Constructor<LitElement>>(superClass: T): T & Constructor<{ctx: CtxSpy}> => {
```
Comment (artalar) [inline packages/npm-lit/src/index.ts]:
```suggestion
export const withReatom = <T extends Constructor<LitElement>>(superClass: T): T & Constructor<{ctx: CtxSpy}> => {
```

**Review by artalar — APPROVED** [review]

**Comment by artalar** [comment]
Awesome, thank you a lot!
